src/binaries/Cargo.toml

```diff
@@ -51,10 +51,10 @@ anyhow = "1.0.65"
 clap = { version = "3.2.22", features = ["derive", "env"] }
 openraft = { git = "https://github.com/datafuselabs/openraft", tag = "v0.7.3" }
 sentry = "0.27.0"
-serde = { version = "1.0.144", features = ["derive"] }
+serde = { version = "1.0.145", features = ["derive"] }
 serde_json = "1.0.85"
 tokio-stream = "0.1.10"
-tonic = "0.7.2"
+tonic = "0.8.1"
```

Comment:
Use `0.8` instead?

Reply:
I would like the version of tonic to be deterministic, there may be compilation issues if the settings are too flexible (the same applies to prost).

Reply:
`0.8.1` is not deterministic. In fact, `Cargo.toml` can't determine anything. We should maintain `Cargo.lock` always.

Reply:
OK, will address in another PR.